presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for operations on timestamps in certain time zones (:issue:`11891`).
+* Fix correctness issue for :func:`array_intersect` and :func:`array_distinct` when input contains
+  both zeros and nulls.
+* Fix ``count(*)`` aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve the table scan performance for structured types.
+* Improve performance for :func:`array_intersect`.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function.
+* Add an optimizer rule to filter the window partitions before executing the window operators.
+* Remove ``ON`` keyword for :doc:`/sql/show-stats`.
+* Restrict ``WHERE`` clause in ``SHOW STATS`` to filters that can be pushed down to the connectors.
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Return final results to clients immediately for failed queries.
+* Performance improvement on ARRAY_INTERSECT.
+
+Web UI
+------
+
+* Fix the rendering of live plan view for queries involving index joins.
+
+Hive Connector Changes
+----------------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix a corner case where the ORC writer fails with integer overflow when writing
+  highly compressible data using dictionary encoding (#11930).
+* Fail queries reading Parquet files if statistics in those Parquet files are
+  corrupt (e.g., min > max). To disable this behavior, set the configuration
+  property ``hive.parquet.fail-on-corrupted-statistics``
+  or session property ``parquet_fail_with_corrupted_statistics`` to false.
+* Add support for :ref:`s3selectpushdown`, which enables pushing down
```

Comment:
When I build the release notes this item gets rendered like below, which is not what we want. 

```
Add support for Amazon S3 Configuration, which enables pushing down projections and predicates into S3 for text files.
```

To fix this we can provide a custom text for the link: 

```
:ref:`S3 select pushdown <s3selectpushdown>`
```